52/souhailshiv/Pomodor.py

```diff
@@ -0,0 +1,65 @@
+import os
+from time import *
+import beepy
+import keyboard
+from tkinter import *
+from tkinter import ttk
+
+root = Tk()
+
+w = Label(root, text="this is pomodor ", fg="red",
+          bg="black", font=("Helevetica", 20))
+w.pack()
+
+
+def pomodor(min):
+    print("work time")
+    sec = 0
+    while min != 0 or sec != 0:
+        print("minuts: ", min, " ", "seconds: ", sec)
+        sleep(1)
+        if sec == 0:
+            if min != 0:
+                min = min-1
+                sec = 59
+            else:
+                break
+        elif keyboard.is_pressed("s"):
+            exit()
+        else:
+            sec = sec-1
+
+    beepy.beep(sound=5)
+    Break(xy)
+
+
+def Break(minut):
+    print("break time")
+    second = 0
+    while True:
```

Comment:
seems this code is duplicated, abstract in helper function

Reply:
what do you mean

Reply:
`Break` and `pomodor` have duplicated code which can be (and should be) prevented by abstracting the duplicated code into a separate method - more info: https://refactoring.guru/smells/duplicate-code